src/views/components/LoginForm/__tests__/container.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import LoginForm, { handleSubmit } from '../container'
+
+describe('LoginForm to match snapshot', () => {
+  it('with default props', () => {
+    const container = shallow(<LoginForm />)
+    const props = {
+      onSubmit: jest.fn()
+    }
+
+    expect(
```

Comment:
Check business logic, not snapshot. You should check that `onSubmit` was called with arguments

Reply:
I replace this in another place and test normally